recipe/recipe.yaml

```diff
@@ -2,18 +2,18 @@ schema_version: 1
 
 context:
   name: azure-batch
-  version: 14.2.0
+  version: 15.0.0b2
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure-batch-${{ version }}.tar.gz
-  sha256: c79267d6c3d3fe14a16a422ab5bbfabcbd68ed0b58b6bbcdfa0c8345c4c78532
+  url: https://files.pythonhosted.org/packages/30/f5/8eb6c79cfc5df4081fdd11cc7c731ab0a7cfb196a6af07a83453b5b3ce43/azure_batch-${{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure-batch-${{ version }}.tar.gz
```

Reply:
@janjagusch, you marked this as resolved but the URL isn't yet fixed.